docs/cubic/cubic.md

```diff
@@ -241,20 +241,22 @@ The steps to do this are as follows:
 
 1.  `git config --global user.name <username>`
 2.  `git config --global user.email <email>`
-3.  Create fine-grained PAT on GitHub.
+3.  `git config --global credential.helper store`
+    -   This will store the PAT in ~/.git-credentials.
+        It's plain text, but not an issue since we only have one repo.
+4.  `git clone https://github.com/PennLINC/affective-instability.git`
```

Comment:
Good chance to make this generic.
```suggestion
4.  `git clone https://github.com/PennLINC/<repo>.git code`
    - This will clone the repository into a new folder named "code".
```